SWIPs/swip-adaptive-node-protocol.md

```diff
@@ -0,0 +1,131 @@
+---
+SWIP: 12
+title: Adaptive nodes capabilities protocol
+author: Louis Holbrook <dev@holbrook.no> (https://holbrook.no), Tim Bansemer (editorial) @FantasticoFox <tim@ethswarm.org>
+status: Draft
+type: Track Networking
+created: 2019-07-11
+---
+
+## Abstract
+
+Adaptive node operation means a node should be able to change the capabilities it has according to changes in available resources.
```

Comment:
I'd specify, "available resources, such as bandwidth, connectivity ,...". This helps guide the reader what it's about